bin/docpub.js

```diff
@@ -10,18 +10,38 @@
 
 const program = require('commander');
 const pkg = require('../package.json');
-const DocpubPipeline = require('../lib/docpub');
+const Docpub = require('../lib/docpub');
 
 program
     .version(pkg.version)
-    .option('-v --verbose', 'verbose output')
-    .option('-p --path <path>', 'path to documents directory (defaults to process directory)');
+    .option('--doc-path <docPath>', 'path to documents directory (defaults to process directory)')
+    .option('--config-path <configPath>', 'path for the config')
+    .option('-v --verbose', 'verbose output');
```

Comment:
Needs a comma: `-v, --verbose`. The default options by commander use a comma, such as `-h, --help`.